index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h3>Hello Microverse</h3>
```

Comment:
Checking the project requirements, this heading was supposed to be an h1. Would you kindly change it to H1?, Having the main heading of your website as an h1, is a best practice as it helps with SEO.